commitlint.config.js

```diff
@@ -0,0 +1,12 @@
+const angularConfig = require('@commitlint/config-angular')
+module.exports = {
+  rules: {
+    ...angularConfig.rules,
+    'type-enum': [
+      2,
+      'always',
+      // Also allow the "chore" commit type.
+      ['chore'].concat(angularConfig.rules['type-enum'][2])
```

Comment:
nit: Is it a temporary override to get the first build passing or do you plan to keep it after as well? Can you add a TODO if former?

Reply:
I see that in the Karma repos we commonly use the `chore` commit type and that type is not allowed in the [Angular Shared Config](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-angular#type-enum) for CommitLint.  Do you think we should allow it indefinitely? Otherwise I wonder what should the commit type of https://github.com/karma-runner/karma-chrome-launcher/pull/249/commits/56004df91a3e23a8a62ded02a12b8fc930ba047f be.

Reply:
I think it's only Semantic Release which uses `chore` type since we have updated the configuration.

According to [the guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) the commit should be `build` - "Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)".

I don't really have a strong opinion on that having or not having `chore`, just thought that it maybe be good to have it consistent accross repos and having less overrides should help with that.

Reply:
Right! We should aim for consistency. Therefore, we should disallow chore in the [core repo](https://github.com/karma-runner/karma) as well. Right?

Reply:
> Right! We should aim for consistency. Therefore, we should disallow chore in the [core repo](https://github.com/karma-runner/karma) as well. Right?

I created https://github.com/karma-runner/karma/issues/3772 to follow up there. I will remove `chore` in this repo in the meanwhile!